Source/WebCore/platform/graphics/texmap/BitmapTextureGL.cpp

```diff
@@ -162,66 +162,26 @@ void BitmapTextureGL::updateContents(Image* image, const IntRect& targetRect, co
     updateContents(imageData, targetRect, offset, bytesPerLine);
 }
 
-static unsigned getPassesRequiredForFilter(FilterOperation::Type type)
-{
-    switch (type) {
-    case FilterOperation::Type::Grayscale:
-    case FilterOperation::Type::Sepia:
-    case FilterOperation::Type::Saturate:
-    case FilterOperation::Type::HueRotate:
-    case FilterOperation::Type::Invert:
-    case FilterOperation::Type::Brightness:
-    case FilterOperation::Type::Contrast:
-    case FilterOperation::Type::Opacity:
-        return 1;
-    case FilterOperation::Type::Blur:
-    case FilterOperation::Type::DropShadow:
-        // We use two-passes (vertical+horizontal) for blur and drop-shadow.
-        return 2;
-    default:
-        return 0;
-    }
-}
-
-RefPtr<BitmapTexture> BitmapTextureGL::applyFilters(TextureMapper& textureMapper, const FilterOperations& filters, bool defersLastFilter)
+RefPtr<BitmapTexture> BitmapTextureGL::applyFilters(TextureMapper& textureMapper, const FilterOperations& filters, bool defersLastFilterPass)
 {
     if (filters.isEmpty())
         return this;
 
     TextureMapperGL& texmapGL = static_cast<TextureMapperGL&>(textureMapper);
     RefPtr<BitmapTexture> previousSurface = texmapGL.currentSurface();
-    RefPtr<BitmapTexture> resultSurface = this;
-    RefPtr<BitmapTexture> intermediateSurface;
-    RefPtr<BitmapTexture> spareSurface;
-
-    m_filterInfo = FilterInfo();
+    RefPtr<BitmapTexture> surface = this;
```

Comment:
The CI pointed out that here is a style error.
But according to the [code style guideline](https://webkit.org/code-style-guidelines/#names-protectors-this), "Ref and RefPtr objects meant to protect this from deletion should be named “protectedThis”.", and I don't think it's applicable here.

Should I address all the CI errors, or can we proceed with some if they’re reasonable?

Reply:
You can ignore it.